test/onnx/test_fx_op_consistency.py

```diff
@@ -1559,6 +1555,18 @@ def skip_torchlib_forward_compatibility(
             "Reshape", "empty tensor"
         ),
     ),
+    skip(
```

Comment:
Can we use xfail? Only use skip for non-deterministic errors.

Reply:
If it is partially failing some subtests, there are decorators that partially xfails based on test inputs.